From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Thomas Petazzoni <thomas.petazzoni@bootlin.com>
Date: Tue, 6 Feb 2024 22:46:59 +0100
Subject: [PATCH] configure.ac: move PY_STDLIB_MOD_SET_NA further up

We will need PY_STDLIB_MOD_SET_NA in next patches further up in the
configure.ac script.

Upstream: N/A

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@bootlin.com>
[ Vincent Fazio: ported to Python 3.13.2 ]
Signed-off-by: Vincent Fazio <vfazio@gmail.com>
[ Bernd Kuhls: ported to Python 3.15.0]
Signed-off-by: Bernd Kuhls <bernd@kuhls.net>
---
 configure.ac | 13 ++++++-------
 1 file changed, 6 insertions(+), 7 deletions(-)

diff --git a/configure.ac b/configure.ac
index 9752d431cba..25a40069fa2 100644
--- a/configure.ac
+++ b/configure.ac
@@ -95,6 +95,12 @@ AC_DEFUN([PY_CHECK_EMSCRIPTEN_PORT], [
   AS_VAR_POPDEF([py_libs])
 ])
 
+# stdlib
+AC_DEFUN([PY_STDLIB_MOD_SET_NA], [
+  m4_foreach([mod], [$@], [
+    AS_VAR_SET([py_cv_module_]mod, [n/a])])
+])
+
 AC_SUBST([BASECPPFLAGS])
 if test "$srcdir" != . -a "$srcdir" != "$(pwd)"; then
     # If we're building out-of-tree, we need to make sure the following
@@ -8092,13 +8098,6 @@ if test -n "$_PYTHREAD_NAME_MAXLEN"; then
 fi
 AC_SUBST([_PYTHREAD_NAME_MAXLEN])
 
-
-# stdlib
-AC_DEFUN([PY_STDLIB_MOD_SET_NA], [
-  m4_foreach([mod], [$@], [
-    AS_VAR_SET([py_cv_module_]mod, [n/a])])
-])
-
 # stdlib not available
 dnl Modules that are not available on some platforms
 AS_CASE([$ac_sys_system],
-- 
2.47.3

